org.keshavnrj.wonderwall.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: org.keshavnrj.wonderwall
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.11'
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: wonderwall
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+  - --filesystem=xdg-download
+  - --device=dri
+  
+  - --talk-name=org.gtk.vfs
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=org.freedesktop.FileManager1
+
+  #- --filesystem=/var/lib/lightdm-data:rw
+
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
```

Comment:
```suggestion
```

This is not needed at all

Reply:
I don't know what each of these lines do in flatpak but, my application need to call gsettings to set wallpaper on gnome based desktop environments. I found a wallpaper app in flathub that uses these lines, I also tested removing these lines and my app was not able to set wallpaper.